x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_color_property.js

```diff
@@ -220,7 +233,17 @@ export class DynamicColorProperty extends DynamicStyleProperty {
     }
 
     mbStops.push(defaultColor); //last color is default color
-    return ['match', ['to-string', ['get', this._options.field.name]], ...mbStops];
+    return ['match', ['to-string', ['get', this._field.getName()]], ...mbStops];
+  }
+
+  _getMbOrdinalColorStops() {
```

Comment:
This method was deleted in https://github.com/elastic/kibana/pull/59953 and is no longer used/needed. Please delete.